stories/Components/MegaMenu/MegaMenu.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import "./megamenu.scss"
+
+export function MegaMenu() {
+  const sections = [
+    {
+      title: 'Section 1',
+      bannerHeading: 'Analytics by country',
+      bannerDescription: 'Gaze upon statistics in wonder...',
+      items: [
+        {
+          title: 'Item 1',
+          subItems: Array(30).fill({ title: 'Sub-item 1', url: '#'})
+        },
+        {
+          title: 'Item 2',
+          subItems: Array(20).fill({ title: 'Sub-item 2', url: '#'})
+        }
+      ]
+    },
+  ]
+
+  const [section, setSection] = useState(null);
+  const [itemIndex, setItemIndex] = useState(0);
+
+  return (
+    <div 
+      className="mg-mega-wrapper"
+      // onMouseLeave={() => setsection(null)}
```

Comment:
I guess this is why we can't deselect a menu item? A delay on mouseout of, hot take, 600ms, would be good.